Remove files from datapacks supported

tcoupland opened the pull request.

mrchrisadams commented on src/cljs/witan/ui/services/analytics.cljs:
```diff
@@ -7,8 +7,6 @@
   (:require-macros [cljs-log.core :as log]
                    [witan.ui.macros :as wm]))
 
-(def analytics-state (atom {}))
```
I'm not too familiar with clojure atoms yet, but I'm assuming we no longer care about the state, becuase we're no longer trying to track if a page change is the same as the previous one.

Anywhere near correct?

acron0 replied:
Bang on :+1:

mrchrisadams commented:
I think I get the general gist:

- the naming convention will be easier to follow in the UI
- we don't need to track state to tell if a page change has occurred anymore

LGTM :+1:

Commit:
Remove files from datapacks supported

tcoupland committed:
Data dash no longer reloads all data on every visit

The app no longer reloads all the data everytime it is routed to the
dashboard. Instead we rely on the inmemory version of the data to be
correct.

When a metadata is opened it is reloaded, hopefully this will avoid
the worst consistency problems that could arise with this solution.
